Analysis/Tutorials/src/outputs.cxx

```diff
@@ -64,7 +64,8 @@ struct DummyTask3 {
 WorkflowSpec defineDataProcessing(ConfigContext const& cfgc)
 {
   return WorkflowSpec{
-    adaptAnalysisTask<DummyTask>(cfgc, TaskName{"task1"}),
-    adaptAnalysisTask<DummyTask2>(cfgc, TaskName{"task2"}),
-    adaptAnalysisTask<DummyTask3>(cfgc, TaskName{"task3"})};
+    adaptAnalysisTask<DummyTask>(cfgc),
+    adaptAnalysisTask<DummyTask2>(cfgc),
+    adaptAnalysisTask<DummyTask3>(cfgc),
```

Comment:
Should we use better names here?

Reply:
In my opinion this tutorial is useless and could be deleted.

Reply:
Fine with me if the OutputObj is illustrated elsewhere